class.wp-cldr.php

```diff
@@ -49,19 +49,51 @@ public function set_locale( $locale ) {
 	*/
 	public function get_CLDR_locale( $wp_locale ) {
 
+		// this array captures the WordPress locales that are signficiantly different from CLDR locales
 		$wp2cldr =  array(
```

Comment:
Are any of these WordPress.com-specific? Do you otherwise anticipate the need for plugin users to override these?  This array might be a good place to do an `apply_filter`